test/framework/src/main/java/org/elasticsearch/logsdb/datageneration/datasource/DefaultMappingParametersHandler.java

```diff
@@ -66,7 +66,7 @@ private Supplier<Map<String, Object>> keywordMapping(
             }
 
             if (ESTestCase.randomDouble() <= 0.2) {
-                injected.put("ignore_above", ESTestCase.randomIntBetween(1, 10000));
+                injected.put("ignore_above", ESTestCase.randomIntBetween(1, 100));
```

Comment:
We don't ever generate such long strings so large values act as a "skip" in a lot of tests.